### PR TITLE
py-html2text: add py39 and py310 subports

### PR DESCRIPTION
**Pull request by ferdy**
###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 11.6 20G165
Xcode 13.0 13A233

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-html2text/Portfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ PortGroup           select 1.0
 
 name                py-html2text
 version             2020.1.16
-revision            0
+revision            1
```
Comment (reneeotten) [inline python/py-html2text/Portfile:9]: you don't / shouldn't increase the `revision` when adding a subport

**Review by reneeotten — CHANGES_REQUESTED** [review]
thanks for the PR @ferdy, one comment below

**Comment by kurthindenburg** [comment]
Look OK testing here - please fix Renee's comment.  Also should we just add py310 as well?

File: python/py-html2text/Portfile (hunk 1 above)
Comment (ferdy) [reply python/py-html2text/Portfile:9]: ciao reneee and thank for the comment.
i have just a doubt and I would be grateful if you confirm it.
i was under the impression that if you add/remove lang version you are supposed to increase the revision. i am doing the same for the migration of p5-* portfile from p5.* to p5.34.
if you confirm I don't have for python or whatever other reason I will be happy to do it! 
cheers!

**Comment by ferdy** [comment]
> Look OK testing here - please fix Renee's comment. Also should we just add py310 as well?

i surely can, yet I can not test at the moment since I am on py39 only installation.
if that's not a problem, I will do for sure! 

cheers,

File: python/py-html2text/Portfile (hunk 1 above)
Comment (ferdy) [reply python/py-html2text/Portfile]: thank reneeotten, now it's clear! i amended the pr and resolved this conversation. cheers,
--
ferdy

**Comment by ferdy** [comment]
> > Look OK testing here - please fix Renee's comment. Also should we just add py310 as well?
> 
> i surely can, yet I can not test at the moment since I am on py39 only installation. if that's not a problem, I will do for sure!

done, pr amended. thank you very much! 
--
ferdy

**Comment by ferdy** [comment]
apologies. closed due to blonde moment.

**Review by reneeotten — APPROVED** [review]

**Comment by reneeotten** [comment]
@kurthindenburg okay to merge?

**Comment by kurthindenburg** [comment]
Thanks, I tested py310 yesterday so it should work OK

**Pull request merged**